### PR TITLE
More documentation for mathematical algorithms.

### PR DESCRIPTION
**Pull request by dysoco**
Some docs I wrote a couple of days ago and forgot to send.

### CODE REVIEW

File: code/mathematical-algorithms/catalan_number/README.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
+# Catalan Number
+The Catalan numbers are a sequence of natural numbers that have applications in the area of combinatorial mathematics.
+
+The firsts Catalan numbers are: 1, 1, 2, 5, 14, 42, 132, 429, 1430...
+
 # cosmos
```
Comment (arnavb) [inline code/mathematical-algorithms/catalan_number/README.md]: Remove this

Comment (dysoco) [reply code/mathematical-algorithms/catalan_number/README.md]: Do you have a suggestion on how to document it? From my understanding the algorithms under catalan_number/ just generate the Catalan numbers; so I though explaining a little bit would be appropriate.

Comment (arnavb) [reply code/mathematical-algorithms/catalan_number/README.md]: No, I meant remove the cosmos part. For the catalan numbers, just post a link to their [Wikipedia](https://en.wikipedia.org/wiki/Catalan_number).

Comment (dysoco) [reply code/mathematical-algorithms/catalan_number/README.md]: Oh right, hadn't noticed that. Thanks.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**